libs/community/langchain_community/llms/sambanova.py

```diff
@@ -691,19 +689,27 @@ class SambaStudio(LLM):
         )
     """
 
-    sambastudio_base_url: str = ""
+    sambastudio_base_url: str = Field(
+        default_factory=from_env("SAMBASTUDIO_BASE_URL", defaul="")
+    )
     """Base url to use"""
 
     sambastudio_base_uri: str = ""
     """endpoint base uri"""
 
-    sambastudio_project_id: str = ""
+    sambastudio_project_id: str = Field(
+        default_factory=from_env("SAMBASTUDIO_PROJECT_ID", defaul="")
+    )
     """Project id on sambastudio for model"""
 
-    sambastudio_endpoint_id: str = ""
+    sambastudio_endpoint_id: str = Field(
+        default_factory=from_env("SAMBASTUDIO_ENDPOINT_ID", defaul="")
```

Comment:
Typo should be deafult

```suggestion
        default_factory=from_env("SAMBASTUDIO_ENDPOINT_ID", defaul="")
```